recipes/newrelic/infrastructure/debian.yml

```diff
@@ -183,6 +184,18 @@ install:
             echo 'proxy: {{.HTTPS_PROXY}}' >> /etc/newrelic-infra.yml
           fi
 
+    log_ssl_ciphers:
+      cmds:
+        - |
+          IS_OPENSSL_INSTALLED=$(which openssl | wc -l)
+          IS_SORT_INSTALLED=$(which sort | wc -l)
+          IS_UNIQ_INSTALLED=$(which uniq | wc -l)
+          if [ $IS_OPENSSL_INSTALLED -gt 0 ] && [ $IS_SORT_INSTALLED -gt 0 ] && [ $IS_UNIQ_INSTALLED -gt 0 ]; then
+            echo "Detecting available SSL ciphers..."
+            openssl ciphers -v | awk '{print " - " $2}' | sort | uniq
```

Comment:
@mbruzina this is an interesting idea. I'm not sure the value of sort/uniq is worth adding a constraint and not log this output when any of those tools is not there, they might not be that common on some bare linux.
Maybe an alternative could be to concatenate all in a single line, with the "Detecting available SSL:" message? Just a thought.

Reply:
That can work too...was just grabbing a command from a recent slack conversation.  There are tons of dupes without sort/unique though.  Let me see what I can do using the utils we bundle with the CLI 👍

Reply:
Sure, dont feel like you have to change or spend a lot of time doing differently. We could always make more changes if we see we're missing most of the usages.